chat-plugins/info.js

```diff
@@ -531,8 +531,12 @@ var commands = exports.commands = {
 						if (!template.learnset) template = Tools.getTemplate(template.baseSpecies);
 						if (!template.learnset) continue;
 						var fullLearnset = template.learnset;
-						while (template.prevo) {
-							template = Tools.getTemplate(template.prevo);
+						while (template.prevo || (template.eventPokemon && template.baseSpecies !== template.species)) {
+							if (template.prevo) {
+								template = Tools.getTemplate(template.prevo);
+							} else {
+								template = Tools.getTemplate(template.baseSpecies);
+							}
 							for (var move in template.learnset) {
 								if (!fullLearnset[move]) fullLearnset[move] = template.learnset[move];
```

Comment:
We're overwriting the `.learnsets` value in the dex.

If we do that, we've made `checkLearnsets`'s algorithm invalid. So `checkLearnsets` needs to be rewritten.